plugins/obs-ffmpeg/obs-ffmpeg-hls-mux.c

```diff
@@ -331,6 +331,7 @@ struct obs_output_info ffmpeg_hls_muxer = {
 	.id = "ffmpeg_hls_muxer",
 	.flags = OBS_OUTPUT_AV | OBS_OUTPUT_ENCODED | OBS_OUTPUT_MULTI_TRACK |
 		 OBS_OUTPUT_SERVICE,
+	.protocols = "HLS",
```

Comment:
As described in voice discussions, I personally believe that using uppercase for these values does not align with common lowercase OBS implementations like the `.encoded_video_codecs` field. While I understand the desire to use the strings from this value directly in the UI, I personally believe that it would be better to uppercase convert these values in the UI and use a hash table if we someday get a protocol which has mixed case.

As of this comment, I'm not aware of any protocols which we officially support which use mixed case: `RTMP, RTMPS, SRT, RIST, FTL, HLS, WHIP`

Reply:
I personally believe that using uppercase for these values does not align with common lowercase OBS implementations like the .encoded_video_codecs field.

If you check #8090, you will see that the whole lowercase/uppercase situation on codecs is not aligned.

Reply:
I was drawing a parallel to codecs in regards to them being all lowercase, even though when presented to the user they are given in their correctly-capitalized form. I was not using it as an example of the uppercase-convert suggestion.